### DISCUSSION TITLE
recaptcha turn of or get a good (free) key.

### DISCUSSION THREAD

**Discussion by waardd** [apache/airflow#39391] [Q&A]
As a fallback we have the option to switch to AUTH_DB with registration.
But the registration is workiong with recaptcha.
Is there a way to turn this off and if not where do i get a good key for it?

**Comment by Taragolis** [comment]
> Is there a way to turn this off

This provided by FAB (Flask-APP Builder) through Flask-WTF  and as soon as I could see there is no way to disable it: https://github.com/dpgaspar/Flask-AppBuilder/issues?q=is%3Aissue+reCaptcha

There is quite popular suggestion from the Internet to set `RECAPTCHA_ENABLED = False` but it doesn't work because neither [FAB](https://flask-appbuilder.readthedocs.io/en/latest/config.html) and [Flask-WTF](https://flask-wtf.readthedocs.io/en/1.2.x/config/#recaptcha) do not have such configuration.

So right now it is not possible to disable it. Maybe in future when we get rid of FAB in Airflow

> where do i get a good key for it?

- https://www.google.com/recaptcha/admin/create
- https://developers.google.com/recaptcha
